metricsrouterv3/metrics_router_v3.go

```diff
@@ -0,0 +1,1883 @@
+/**
+ * (C) Copyright IBM Corp. 2022.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * IBM OpenAPI SDK Code Generator Version: 3.61.0-1667892a-20221109-194550
+ */
+
+// Package metricsrouterv3 : Operations and models for the MetricsRouterV3 service
+package metricsrouterv3
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"reflect"
+	"time"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	common "github.com/IBM/platform-services-go-sdk/common"
+	"github.com/go-openapi/strfmt"
+)
+
+// MetricsRouterV3 : IBM Cloud Metrics Router allows you to configure how to route platform metrics in your account.
+//
+// API Version: 3.0.0.preview
+type MetricsRouterV3 struct {
+	Service *core.BaseService
+}
+
+// DefaultServiceURL is the default URL to make service requests to.
+const DefaultServiceURL = "https://au-syd.metrics-router.dev.cloud.ibm.com"
```

Comment:
This was generated as the default service URL because it was the first entry in the "servers" field within the API definition.  Is this the desired default endpoint for the service?  If not, then please adjust the order of the entries within the "servers" field so that the entry containing the default endpoint URL is the first entry in the list.

Reply:
For this GA we are supporting only au-syd, so the DefaultServiceURL is good for now.